src/python/Dockerfile

```diff
@@ -91,4 +91,5 @@ ENV PANTS_LOCAL_STORE_DIR=/mnt/pants-cache/lmdb_store
 ENV PANTS_SETUP_CACHE=/mnt/pants-cache/setup
 
 # /mnt/grapl-root is mounted by `integration-tests.nomad` at runtime.
-ENTRYPOINT ["/bin/bash", "-o", "errexit", "-o", "nounset", "-c", "cd /mnt/grapl-root && ./pants filter --filter-target-type='python_tests' :: | xargs ./pants --tag='-needs_work' test --pytest-args='-m integration_test'"]
+# -rA reports stdout/stderr for all tests, not just failed ones.
+ENTRYPOINT ["/bin/bash", "-o", "errexit", "-o", "nounset", "-c", "cd /mnt/grapl-root && ./pants filter --filter-target-type='python_tests' :: | xargs ./pants --tag='-needs_work' test --pytest-args='-m integration_test -rA'"]
```

Comment:
Rather than using a `cd`, we should specify a `WORKDIR` in the Dockerfile. Additionally, once your entrypoint starts to get this complex (and just plain _long_), it should probably be a standalone script that is executed. If we took that route, we could get rid of all the `/bin/bash -o errexit -o nounset -c` stuff. The default shell for Linux in Dockerfiles is `/bin/sh -c`, which is certainly all that is needed for executing a script; the only reason we set such an entrypoint in our old docker-compose files was because we were overriding very non-trivial commands, and that was the most convenient and proper way of doing that. What we're doing here is a very different scenario.

In fact, I think you could just leave out `ENTRYPOINT` altogether and set `CMD` to `/custom-script-that-runs-the-tests.sh` and call it a day.

Also, are there long options for `-r` and `-A` that are more self-documenting?